### ISSUE TITLE
Error in fcn8s_pascal.yml

### ISSUE THREAD

**Issue by xhoarau** [meetshah1995/pytorch-semseg#265]
I have seen an error in the file fcn8s_pascal.yml, in the selection of the dataset.
For the val split it select "val" instead of "train_aug_val" this leads to 400 images that are both in training and validation set.
After correction, the accuracy of the FCN8s network fall from 65% to 57%.
I think it is important to correct it, and do you know how to get an implementation that achieve the same as the paper?
Thank you in advance

**Comment by hjhjb** [comment]
我已收到，谢谢
